config.yaml

```diff
@@ -30,3 +30,11 @@ options:
   mysql-interface-database:
     description: "The database name for the legacy 'mysql' relation"
     type: "string"
+# Experimental features
+  experimental-max-connections:
```

Comment:
I'd like to challenge the config option name. I totally understand that it's experimental and it could be removed at any point so clarifying it in the config description is fine.

However, adding the prefix `experimental-` to the config option itself is a little bit awkward. Because the max-connections in MySQL itself is a quite known config and it's not experimental or anything like that. The idea behind the config naming is to indicate that "this option could disappear anytime once the charm gets capable to make the best decision" so I prefer a different name something like `max-connections-override` or something like that to indicate "while charm will try to make the best decision to configure max connections automatically, you can still override the value explicitly".

Reply:
@nobuto-m, Thank you for your feedback!
We have decided to use the `experimental-` prefix on a Product level to make it consistent across multiple solutions. This decision affects not only `max-connections` but also a few other things where `override` doesn't work.
Please provide product feedback or discuss directly with decision-makers.